### PR TITLE
Include requirements.txt in setup.py

### PR DESCRIPTION
**Pull request by adivijaykumar**
`setup.py` doesn't take in the requirements (!). This means that the installed package from pypi won't really work well.

### CODE REVIEW

**Comment by adivijaykumar** [comment]
@prayush @vaishakp any idea why the tests are failing? They are failing on master too with an import error,

**Comment by vaishakp** [comment]
> @prayush @vaishakp any idea why the tests are failing? They are failing on master too with an import error,

Checking

**Comment by vaishakp** [comment]
> > @prayush @vaishakp any idea why the tests are failing? They are failing on master too with an import error,
> 
> Checking

This was due to a minor bug in wftools. On merging the PR #26, things should be fine.

**Comment by prayush** [comment]
@adivijaykumar is there anything holding up this PR now?

File: requirements.txt [hunk 1]
```diff
@@ -6,5 +6,5 @@ lalsuite
 ligo-common
 pycbc
 pandas
-sxs[ecosystem]
+sxs
```
Comment (adivijaykumar) [inline requirements.txt:9]: @prayush this PR is ready for merge. However, I wanted to check with you if changing `sxs[ecosystem]` to `sxs` in the requirements is okay. I made this change to reduce the total number dependencies (although I guess `pycbc` and `lalsuite` are contributing most of them). The tests don't really fail due to this.

Comment (prayush) [reply requirements.txt:9]: Yes, that is actually good to reduce. `sxs[ecosystem]` adds things like jupyter, matplotlib, etc, which are strictly not needed for this package to function.

**Pull request merged**